modules/programs/getmail.nix

```diff
@@ -0,0 +1,59 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  accounts = filter (a: a.getmail.enable)
+    (attrValues config.accounts.email.accounts);
+
+  renderAccountConfig = account: with account;
+    let
+      passCmd = concatMapStringsSep ", " (x: "'${x}'") passwordCommand;
+      renderedMailboxes = concatMapStringsSep ", " (x: "'${x}'") getmail.mailboxes;
+      retrieverType = if imap.tls.enable
+        then "SimpleIMAPSSLRetriever"
+        else "SimpleIMAPRetriever";
+      destination = if getmail.destinationCommand != null
+        then
+        {
+          destinationType = "MDA_external";
+          destinationPath = getmail.destinationCommand;
+        }
+        else
+        {
+          destinationType = "Maildir";
+          destinationPath = maildir.absPath;
```

Comment:
shouldn't it be `destinationPath = maildir.absPath + "/${accout.name}/"; otherwise all accounts share the same destinationPath